SpringProject/src/main/resources/db/migration/V2__add_test_data.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO conference(conference_name, conference_date_time, city, conference_description, conference_topic)
```

Comment:
I don't think test data is something you should add as a migration, migrations should only set up tables without adding any data. Later on you might have some automation in place so that when you run tests, you automatically insert test data and clear it afterwards but for now I would only add data manually (by running this SQL in the workbench).